examples/kubeval/README.md

```diff
@@ -0,0 +1,26 @@
+# Kubeval
+
+The `kubeval` KRM config function validates Kubernetes resources against their
+Kubernetes OpenAPI definition using kubeval. This example invokes the kubeval
+function against Kubernetes v1.18.0 using declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/kubeval .
+kpt fn run kubeval --network
+```
+
+## Expected Results
+
+The function outputs the following error:
+
+```sh
+[ERROR] Invalid type. Expected: [integer,null], given: string in object 'v1/ReplicationController//bob' in file configs/example-config.yaml
```

Comment:
Why isn't the path to the problematic field part of the error message?

This example has `spec.replicas` in the output message:

https://www.kubeval.com/#json

Reply:
This is related to https://github.com/GoogleContainerTools/kpt/issues/719. When kpt prints results to console, it doesn't print the full results. I will have this blueprint use `--results-dir` and the results file clearly shows the error comes from field path `spec.replicas`.

Reply:
Why does omitting tags impact this? `spec.replicas` is part of field path not a tag.

Reply:
I updated the issue. Kpt only includes the following fields to output to the console: message, severity, resourceRef, and file path. Basically everything else is omitted. As a result we need to add `--results-dir` and view the yaml for the full results.

Reply:
Included the full results which show `spec.replicas`